pkgs/development/tools/build-managers/redo-apenwarr/default.nix

```diff
@@ -1,27 +1,16 @@
-{ stdenv, lib, python27, fetchFromGitHub, mkdocs, which, findutils, coreutils
+{ stdenv, lib, python3, fetchFromGitHub, mkdocs, which, findutils, coreutils
 , perl
 , doCheck ? true
-}: let
-
-  # copy from 
-  # pkgs/applications/networking/pyload/beautifulsoup.nix
-  beautifulsoup = python27.pkgs.callPackage ./beautifulsoup.nix {
-    pythonPackages = python27.pkgs;
-  };
-
-  mkdocs-exclude = python27.pkgs.callPackage ./mkdocs-exclude.nix {
-    pythonPackages = python27.pkgs;
-  };
-in stdenv.mkDerivation rec {
+}: stdenv.mkDerivation rec {
 
   pname = "redo-apenwarr";
-  version = "0.42";
+  version = "20200304";
```

Comment:
version [0.42a](https://github.com/apenwarr/redo/releases) has just been released. Maybe this is better than going to unstable?

Reply:
What a coincidence. Thanks, I updated the version, sha256 is still the same.